modules/agentless_scanner_integration/main.tf

```diff
@@ -0,0 +1,105 @@
+module "agentless-resources-us-east-1" {
```

Comment:
who do we have region information here? Please push provider and region information out of here.